server/coordinator/procedure/scatter.go

```diff
@@ -49,39 +49,43 @@ func scatterPrepareCallback(event *fsm.Event) {
 
 	waitForNodesReady(c)
 
-	if c.GetClusterState() == clusterpb.ClusterTopology_STABLE {
+	if c.GetClusterState() == storage.Stable {
```

Comment:
The meaning of this variable `storage.Stable` is not clear, change it to `storage.ClusterState_State`